docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -74,11 +71,11 @@ It's basically the same, to keep it familiar –
 we're keeping the geometrical shape, to resonate with the _building blocks_ idea of the components –
 however, we're reducing the emphasis on the 3D perspective, stepping away from the notion of elevation that Material Design coined.
 
-<img loading="lazy" src="/static/blog/material-ui-is-now-mui/old-new.png" alt="Side-by-side of the new and old logo" style="width: 612px; margin-bottom: 16px;" />
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/old-new.png" alt="Side-by-side of the new and old logo" style="width: 796px; margin-top: 16px; margin-bottom: 16px;" />
```

Comment:
I personally struggle a bit to read the text 'Old', New', MUI X I think that it's a combination of a pixel resolution issue (it's blurry) + the font size being maybe a bit too small. It's better in https://mui.com/blog/material-ui-is-now-mui/#new-logos